docs/advanced/dao/01-creation.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Creation
+sidebar_label: Creation
+sidebar_position: 1
+---
+
+# Dao Creation
+
+Two framework contracts manage the `DAO` contract creation process:
+
+- The [`DAOFactory`](docs/osx/reference-guide/framework/dao/DAOFactory).
+- The [`DAORegistry`](docs/osx/reference-guide/framework/dao/DAORegistry).
+
+The `DAOFactory` is the contract that creates and sets up a `DAO` for you. The `DAORegistry` is used by the `DAOFactory` to register the DAOs that are created. Plugins are installed in the DAO by the `PluginSetupProcessor`.
+
+![Creation Schema](/img/advanced/dao/aragon-osx-dao-creation.drawio.svg)
+
+## `DAOFactory`
+
+The `DAOFactory` creates and sets up a `DAO` for you in four steps with the `createDao` function. The function requires the `DAOSettings` including:
+
+- The trusted forwarder address for future [ERC-2771 (Meta Transaction)](https://eips.ethereum.org/EIPS/eip-2771) compatibility that is set to `address(0)` for now
+- The ENS name (to be registered under the `dao.eth` domain)
+- The [ERC-4824 (Common Interfaces for DAOs)](https://eips.ethereum.org/EIPS/eip-4824) `daoURI`
+- Optional metadata
+
+The DAO also requires an array of `PluginSettings` containing `PluginSetup` and its respective setup data for the plugins to be installed in the DAO.
+
+When the `createDao` function is called in the `DAOFactory` this triggers a four step process for creating a DAO:
+
+1. Creates a new DAO by deploying an [ERC-1967](https://eips.ethereum.org/EIPS/eip-1967) proxy pointing to the latest Aragon OSx `DAO` implementation and becomes the initial owner.
+2. Registers the new contract in the [`DAORegistry`](docs/osx/how-it-works/framework/dao-creation/#daoregistry).
+3. Installs the plugins using the `PluginSetupProcessor` (see also the section about [the plugin setup process](docs/osx/how-it-works/framework/plugin-management/plugin-setup/)).
+4. Sets the [native permissions](docs/osx/how-it-works/core/permissions/#permissions-native-to-the-dao-contract) of the `DAO` and revokes its own ownership.
+
+### Plugins
+
+When calling `createDao` an array of `PluginSettings` are requested. A DAO cannot be created without at least one plugin. The DAO contract works as a permission manager system but it is agnostic to the type of governance that you want to use to manage the DAO. We currently provide two plugins that can be used for governing your DAO:
+
+- [`Multisig`](https://link-to-more-info/)
+- [`TokenVoting`](https://link-to-more-info/)
```

Comment:
Maybe we can come again to this comment when we merge other docs, but I created a dedicated section for governance plugins, maybe could point to it.